captoolkit/ibe/corribe.py

```diff
@@ -360,7 +360,15 @@ def main():
 
         # Subset
         t_ibe = t_ibe[k1:k2]
-        z_ibe = z_ibe[k1:k2, :, :]
+        if (z_ibe.ndim == 4):
```

Comment:
@tsutterley not having time to dig too deep into the code... what is this change for?

Reply:
Oops.  I forgot I made this change (I would have commented about it). The purpose of the change is that in some newer ERA5 fields there can be 4 dimensions. The files that are 4D will have both a mix of reanalysis and preliminary data (ERA5 and ERAST).  Only one of the expver indices of this dimension will be valid at a given time. With the change it simply finds whatever expver slice is valid for a given time index. The change was needed as the code is looking for a 3D variable and not 4D. 

Can read more about expver variables here: https://confluence.ecmwf.int/plugins/servlet/mobile?contentId=173385064#content/view/173385064